docs/src/upgrade/release-3.7.x.asciidoc

```diff
@@ -30,9 +30,10 @@ complete list of all the modifications that are part of this release.
 === Upgrading for Users
 
 ==== String Manipulation Steps
-Modifications to the concat() step as well as additional String manipulation steps have been added to this version.
+This version introduces the following new string manipulation steps `asString()`, `length()`, `toLower()`, `toUpper()`, `trim()`,
+`lTrim()`, `rTrim()`, `reverse()`, `replace()`, `split()`, and `substring()`, as well as modifications to the `concat()` step introduced in 3.7.0.
```

Comment:
+ format()

Reply:
Format is in its own section, do we want to add it here?